server/index.js

```diff
@@ -16,6 +16,7 @@ const fs = require('fs');
 const { setupExpressRoute: setRouteForPDF } = require('frappejs/server/pdf');
 const auth = require('./../auth/auth')();
 const morgan = require('morgan')
+const posRegister = require('./file');
```

Comment:
file is not related to pos .. 

you can rename it to `setupFileMethods`

Reply:
oops!,these are the naming before I moved to frappes earlier I had them in accounting/pos.